source/includes/api/auth/_status.md.erb

```diff
@@ -67,9 +67,9 @@ The auth's credentials are no longer valid and need to be reauthenticated using
 
 ![Bill Authenticate](<%= config.s3 %>/bill-authenticate.png)
 
-### hold 
+### hold
 
-For a number of reasons an auth can have the status of `hold`. When an auth is on hold we'll present you with a `reason` that you in turn can present to your users. To remove this status, the user must take the appropriate action as described in the `reason`. This usually involves them needing to login into their online account. Once the user has completed the action, make a request to the `/auth/{authId}/fix-hold` endpoint to resume their services.
+For a number of reasons an auth can have the status of `hold`. When an auth is on hold we'll present you with a `reason` that you in turn can present to your users. To remove this status, the user must take the appropriate action as described in the `reason`. This usually involves them needing to login into their online account. Once the user has completed the action, make a request to the `/auth/{authId}/fix-hold` endpoint to resume their services. In addition a [code](#auth-hold-codes) is provided for handling the hold programmatically if required.
```

Comment:
An auth's status can be `hold` for several reasons. When an auth is on hold, the response contains a `reason` for the hold that you can present to your users. To fix the hold, the user must perform the action described in the `reason`, which usually requires them to log in to their online account. Once the user has completed the action, make a request to the `/auth/{authId}/fix-hold` endpoint to resume their services. 

Additionally, a [code](#auth-hold-codes) is provided for handling the hold programmatically, if required.